src/main/java/com/khutircraftubackend/config/SecurityConfig.java

```diff
@@ -25,7 +25,7 @@
 public class SecurityConfig {
 
     private final JwtAuthenticationFilter jwtAuthenticationFilter;
-    private final UserDetailsService userDetailsService;
+    private final UserDetailsServicesImpl userDetailsService;
```

Comment:
why can't we use interface here?
why do we change this config class as part of "docker related story"?

Reply:
its UserDetailsServicesImpl userDetailsService; that's exactly what we use.
Config...